lldb/source/Plugins/ObjectFile/Minidump/ObjectFileMinidump.cpp

```diff
@@ -65,56 +66,53 @@ bool ObjectFileMinidump::SaveCore(const lldb::ProcessSP &process_sp,
   if (!process_sp)
     return false;
 
-  MinidumpFileBuilder builder;
+  llvm::Expected<lldb::FileUP> maybe_core_file = FileSystem::Instance().Open(
+      outfile, File::eOpenOptionWriteOnly | File::eOpenOptionCanCreate);
+  if (!maybe_core_file) {
+    error = maybe_core_file.takeError();
+    return false;
+  }
+  MinidumpFileBuilder builder(std::move(maybe_core_file.get()));
```

Comment:
pass `process_sp` into the constructor, then we don't need to keep passing it to the `MinidumpFileBuilder` methods.